app/processing/payment/calculate-scheduled-payments/get-settlement-value.js

```diff
@@ -1,5 +1,5 @@
 const getSettledValue = (previousSettlements) => {
```

Comment:
We are returning maximum value because that is the pattern of how we store various payments made, in practice what we want is the amount settled so far, hence reason for leaving the name as it was.